packages/vscode-extension/src/controls/SampleGallery.tsx

```diff
@@ -80,15 +81,14 @@ export default class SampleGallery extends React.Component<any, any> {
             <div className="sample-stack">
               <SampleAppCardList
                 samples={this.state.samples}
-                baseUrl={this.state.baseUrl}
                 highlightSample={this.highlightSample}
               />
             </div>
           </div>
         )}
         {this.state.highlightSample != "" && (
           <SampleDetailPage
-            baseUrl={this.state.baseUrl}
+            baseUrl={hightSample.url}
```

Comment:
Does this mean highlightSample? it's a bit confusing.

Reply:
updated